runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -2564,14 +3267,31 @@ TR::Node *TR_VectorAPIExpansion::transformNary(TR_VectorAPIExpansion *opt, TR::T
 
       for (int32_t i = 0; i < numOperands; i++)
          {
-         if (operands[i]->getOpCodeValue() == TR::aload)
+         TR::Node *operand = operands[i];
+
+         if (operand->getOpCodeValue() == TR::aload)
             {
             TR::DataType opCodeType = vectorType;
+            TR::SymbolReference *operandSymRef = operand->getSymbolReference();
+            int32_t operandId = operandSymRef->getReferenceNumber();
 
-            if (opt->_aliasTable[operands[i]->getSymbolReference()->getReferenceNumber()]._objectType == Mask)
+            if (opt->_aliasTable[operandId]._objectType == Mask)
+               {
                opCodeType = TR::DataType::createMaskType(elementType, vectorLength);
+               }
+            vectorizeLoadOrStore(opt, operand, opCodeType);
+            }
+         else if (operand->getOpCodeValue() == TR::acall)
+            {
+            TR::DataType opCodeType = vectorType;
+            TR::SymbolReference *operandSymRef = operand->getSymbolReference();
+            int32_t operandId = operandSymRef->getReferenceNumber();
```

Comment:
You don't need `operandId` and `operandSymRef` in this path, do you?

Reply:
Fixed